LibVLCSharp.Forms.Platforms.GTK/LibVLCSharp.Forms.Platforms.GTK.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net47</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Xamarin.Forms.Platform.GTK" Version="3.2.0.871581" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\LibVLCSharp.Forms\LibVLCSharp.Forms.csproj" />
+    <ProjectReference Include="..\LibVLCSharp.GTK\LibVLCSharp.GTK.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="gdk-sharp">
+      <SpecificVersion>False</SpecificVersion>
```

Comment:
`<SpecificVersion>False</SpecificVersion>` this seem to be removed systematically by VS4Mac FYI.